osu.Game/Rulesets/UI/ModIcon.cs

```diff
@@ -56,15 +62,44 @@ public ModIcon(Mod mod)
                     Icon = OsuIcon.ModBg,
                     Shadow = true,
                 },
+                modAcronym = new OsuSpriteText
+                {
+                    Origin = Anchor.Centre,
+                    Anchor = Anchor.Centre,
+                    Colour = OsuColour.Gray(84),
+                    Alpha = 0,
+
+                    Font = OsuFont.Numeric.With(null, size - 57.5f),
```

Comment:
```suggestion
                    Font = OsuFont.Numeric.With(null, 22f),
```
.5 isn't significant, also prefer to have things defined - would like the `size - 35` below to be simplified to `45` also.